src/BhudeviS/PrintUniqueNumber.java

```diff
@@ -0,0 +1,22 @@
+package BhudeviS;
+
+public class PrintUniqueNumber {
+	void PrintUniqueNumber()
+	{
+		int arr1[]={1,3,6,9,7};
+		int arr2[]={2,5,6,9,8};
```

Comment:
Here what is same dupliucate number comes in array1 or array2 ...

like for this condition...

int arr1[]={1,3,6,9,7};
int arr2[]={2,5,6,6,9,8};

Correct logic go with double for loop...